### ISSUE TITLE
Installation instructions: where to put the files

### ISSUE THREAD

**Issue by bibcoder** [ANSAKsoftware/exifax-testfax#1]
Dear Arthur Klassen,

I just updated Linux, and stumbled upon your hylafax-exim code on github. I wonder, where to put this "testFaxFunc.mysql" file, in which directory? -- maybe you could tell me or add a line explaining this in your github code. 

Would be highly appreciated.

Thanks for your great contribution & kind regards to Canda

**Comment by anklassen** [comment]
Hey hey, bibcoder:

I've tried to address your request [here](https://github.com/ANSAKsoftware/exifax-testfax/pull/2). Does that say what you needed?

**Comment by bibcoder** [comment]
Hey anklassen,
thank you very much for your reply and your additions. I didn't reply yet, even though I now understood it much better, but didn't get it working finally yet, and wanted to try some more, before asking you again. 
It's probably because I never did much with exim and mysql and neither both. I will give it a try this weekend.

**Comment by anklassen** [comment]
Hey hey bibcoder:

Thanks for the reply!

Unless you're configuring an existing gateway between exim and hylafax, this package isn't much use to you. That said, thanks for pushing me to improve the documentation. Given that my "first customer" was a sysadmin who knew what all the parts referred to, I didn't think of Things A Newbie Might Need when I wrote it the first time.

If you ARE trying to tailor it to a non-North American market (which from the times of your posts, I assume you are),
and you'd like some assistance,
and the hints I added for how one might customize testFaxFunc.mysql for the UK are not helpful,

feel free to describe how your phone system works and I'll be glad to send back some further notes. (and yeah, I'm perennially curious about how other phone systems work -- the notes I wrote, for instance, will flat-out not work in Mexico as I last saw its system configured [and may have changed] and they'll be partially helpful/partially not so in Australia for instance)

cheers...ank
